src/main/java/com/kodilla/ecommercee/service/CartService.java

```diff
@@ -55,8 +55,13 @@ public Item getItem(Long itemId) {
         return itemRepository.getOne(itemId);
     }
 
-    public boolean addItem(Item item) {
-       return cartRepository.getOne(item.getCart().getId()).getItems().add(itemRepository.save(item));
+    public Item addItem(Item item) {
+       if(item.getCart().getId().equals(null) || item.getProduct().getId().equals(null)){
+           throw new EntityNotFoundException("Assignments to cart or product were not found");
```

Comment:
Poprawiłam na i wyśle w nastepnym commicie: throw new EntityNotFoundException("Item with assignments to cart or product was not found");